doc/sphinx/source/gensidebar.py

```diff
@@ -59,6 +59,7 @@ def _header(project, text):
     _header("esmvaltool", "ESMValTool")
     _write("esmvaltool", "Introduction", "introduction")
     _write("esmvaltool", "Getting started", "quickstart/index")
+    _write("esmvaltool", "ESMValTool Functionalities", "functionalities")
```

Comment:
I would suggest to move the "ESMValTool Functionalities" upward inbetween "Introduction" and "Getting started".